playbooks/roles/os_temps/tasks/build_new_app.yml

```diff
@@ -24,5 +24,5 @@
   retries: 6
   delay: 10
   ignore_errors: yes
-  when: build_results[template_name]|bool == false
+  when: (build_results[template_name]|bool == false and app_build_status.stdout_lines is defined)
```

Comment:
+1. nit: we wouldn't need (), we can use "" that is when we use () for or condition or something, but again seems to have worked earlier, e.g: ln #12

Reply:
Agreed, I put it that way for sake of uniformity.

Reply:
@skatlapa @dirgim We can create another PR for uniformity on when statements.  What is the standard for that out of curiosity?

Reply:
@arilivigni @dirgim The usual standard(not really official convention) is just dropping the () around when.
e.g: https://ansible-docs.readthedocs.io/zh/stable-2.0/rst/playbooks_conditionals.html
Notice the (foo) or (bar) above in case of "or", that's like the usual convention, but imo it's better to follow conventions as the code is open-source and as good practice in general.

Reply:
Yeah, it would be best to go over the whole project and align when statements with the standard convention in a separate PR. I wouldn't advise radical changes as jinja2 is a bit more moody with expressions than some other languages, but removing the outer brackets should be fine.